daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -199,7 +200,7 @@ object SBuiltin {
   final case object SBDivNumeric extends SBBinaryOpNumeric2(divide)
 
   final case object SBRoundNumeric extends SBuiltin(3) {
-    def execute(args: util.ArrayList[SValue], machine: Machine): Unit = {
+    override def execute(args: util.ArrayList[SValue], machine: Machine): Unit = {
```

Comment:
Other `SBuiltin` implementations use `final def execute`. Should this one be `final` as well?

Reply:
Good catch.